shell/platform/android/io/flutter/embedding/android/AndroidTouchProcessor.java

```diff
@@ -183,6 +187,7 @@ private void addPointerForIndex(
 
     long timeStamp = event.getEventTime() * 1000; // Convert from milliseconds to microseconds.
 
+    packet.putLong(motionEventId.getId());
```

Comment:
do you know if the other embeddings have to be updated as well? I wonder if adding the field to the end is safer.

Reply:
SG